rke/templates/templates.go

```diff
@@ -96,16 +97,19 @@ var TemplateIntroducedRanges = map[string][]string{
 func LoadK8sVersionedTemplates() map[string]map[string]string {
 	return map[string]map[string]string{
 		kdm.Calico: {
-			">=1.22.0-rancher1-1":                    calicov3211,
-			">=1.21.0-rancher1-1 <1.22.0-rancher1-1": calicov319,
-			">=1.20.4-rancher1-1 <1.21.0-rancher1-1": calicov3171,
-			">=1.19.4-rancher1-2 <1.20.4-rancher1-1": calicov3165,
-			">=1.19.0-rancher0 <1.19.4-rancher1-2":   calicov3160,
-			">=1.17.4-rancher0 <1.19.0-rancher0":     calicov117Privileged,
-			">=1.17.0-rancher0 <1.17.4-rancher0":     calicov117,
-			">=1.16.8-rancher0 <1.17.0-rancher0":     calicov117Privileged,
-			">=1.16.4-rancher1 <1.16.8-rancher0":     calicov117,
-			">=1.16.0-alpha <1.16.4-rancher1":        calicov116,
+			">=1.24.9-rancher1-1 < 1.25.2-rancher1-1":  calicov3225,
+			">=1.23.15-rancher1-1 < 1.24.2-rancher1-1": calicov3225,
+			">=1.24.2-rancher1-1 <1.24.9-rancher1-1":   calicov3211,
```

Comment:
1. Swap the order of lines for more logical order of 1.23 followed by 1.24
2. Even though there were no releases earlier than `1.24.2-rancher1-1` I think it's cleaner to use `>=1.24.0-rancher1-1 <1.24.9-rancher1-1` to indicate "any 1.24.x release until and not including 1.24.9-rancher1-1". Same for the range on the other line.
3. Remove unnecessary space after `<` (I'm not sure if it would break anything, but it should be consistent).

```suggestion
			">=1.24.0-rancher1-1 <1.24.9-rancher1-1":   calicov3211,
			">=1.23.15-rancher1-1 <1.24.0-rancher1-1":  calicov3225,
```

Reply:
done